### PR TITLE
lift kube dependency to v0.18.6

### PR DESCRIPTION
**Pull request by humblec**
Considering we are at tip of 3.0.0 release , its better to make use of latest available
minor release of kube major version 1.18. It helps a lot to have better code available
from dependency chain to consume with the release. 

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal.

File: go.mod [hunk 1]
```diff
@@ -29,38 +29,38 @@ require (
...
+	k8s.io/api v0.18.6
+	k8s.io/apimachinery v0.18.6
+	k8s.io/client-go v0.18.6
+	k8s.io/cloud-provider v0.18.6
+	k8s.io/cri-api v0.18.6 // indirect
 	k8s.io/klog/v2 v2.3.0
-	k8s.io/kubectl v0.18.0 // indirect
-	k8s.io/kubernetes v1.18.0
+	k8s.io/kubectl v0.18.6 // indirect
+	k8s.io/kubernetes v1.18.5
```
Comment (Madhu-1) [inline go.mod:39]: why this is 18.5? it should be 18.6 as its the latest release.

Comment (humblec) [reply go.mod:39]: yep

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal

**Pull request merged**